clang/lib/Sema/SemaChecking.cpp

```diff
@@ -4864,14 +4877,22 @@ bool Sema::BuiltinVAStart(unsigned BuiltinID, CallExpr *TheCall) {
     return true;
 
   // Verify that the second argument to the builtin is the last argument of the
```

Comment:
```suggestion
  // Verify that the second argument to the builtin is the last non-variadic argument of the
```

right?  Alternatively 'named', though that isn't really accurate.

Reply:
Yup, I'll fix.